instance/gandi.py

```diff
@@ -136,6 +136,36 @@ def add_dns_record(self, record):
         result = client.execute()
         return result
 
+    def list_dns_records(self, record):
```

Comment:
I'm thinking about this change @swalladge, I can change this, but then it would break the convention we use at setting and removing DNS records (`set_dns_record` and `remove_dns_record`). What are your thoughts about it? Should we still change the `list_dns_records`' signature?

Reply:
Nah that's fine - better to be consistent here.  If we refactor things in the future it would be nice to define the types more clearly.